docs: access control user docs (#696)
bytebase/bytebase.com: content/docs/administration/access-control.md

---

> title: Access Control

Reviewer: Database Access Control

Author: updated.

---

<hint-block type="info">

This feature is only available in the Enterprise plan.

</hint-block>

> By default, instances and databases in a [protected environment](/docs/administration/environment-policy/tier) will be inaccessible to developers from the [SQL Editor](/docs/sql-editor/overview).

Reviewer: ```suggestion
By default, instances and databases in a [protected environment](/docs/administration/environment-policy/tier) will be inaccessible to users with Workspace Developer role from the [SQL Editor](/docs/sql-editor/overview).
```

![editor-inaccessible](/static/docs/administration//access-control/access-control-editor-inaccessible.webp)

> Workspace owners and DBAs can configure some databases in a [protected environment](/docs/administration/environment-policy/tier) as accessible to developers via **Settings / Workspace / Access Control**.

Reviewer: ```suggestion
For databases in the [protected environment](/docs/administration/environment-policy/tier), workspace owners and DBAs can grant database access to workspace developers via **Settings / Workspace / Access Control**.
```

> Click **Add rule**, search and select databases in protected environments.

Reviewer: ```suggestion
Click **Add rule**, search and select databases in the protected environments.
```
